src/main/scala/com/atomist/rug/test/gherkin/handler/AbstractHandlerScenarioWorld.scala

```diff
@@ -71,6 +71,16 @@ abstract class AbstractHandlerScenarioWorld(definitions: Definitions, rugs: Opti
   }
 
   /**
+    * Return the message or null if none was recorded
+    */
+  def message: jsScalaHidingProxy = {
+    planOption match {
+      case Some(p) => p.messages.map(jsScalaHidingProxy.apply(_)).seq(0)
+      case _ => null
```

Comment:
Could we use Option[jsScalaHidingProxy] instead and avoid the null?

Also, I think you can replace `.seq(0)` with `.head`, or ideally `.headOption` :)